synthesizer/src/vm/mod.rs

```diff
@@ -76,6 +80,8 @@ pub struct VM<N: Network, C: ConsensusStorage<N>> {
     atomic_lock: Arc<Mutex<()>>,
     /// The lock for ensuring there is no concurrency when advancing blocks.
     block_lock: Arc<Mutex<()>>,
+    /// A cache containing the list of recent succesfully verified transactions.
+    verified_transactions: Arc<RwLock<LruCache<N::TransactionID, ()>>>,
```

Comment:
Can we use a lazy static instead?

Reply:
Rust doesn't allow generic statics, and we're using generic over the `Network` here

Reply:
Good callout, thanks for pointing that out.